modules/core/src/main/java/org/apache/ignite/internal/processors/cache/ValidationOnNodeJoinUtils.java

```diff
@@ -209,6 +203,44 @@
         return null;
     }
 
+    /**
+     * @param locDesc Local cache descriptor.
+     * @param joinedData Joined node cache data.
+     * @param isGridActive Is grid active.
+     * @param errorMsg Error message builder.
+     */
+    static void validateSqlSchema(
```

Comment:
Let's inline this method as it has only one usage.

Reply:
I think that it is not really good idea here. `validateNode` is too complex and verbose.

Reply:
We can refactor `validateNode` in the next patches.
For now, we should try to keep PR as simple as we can.